src/legacy/core_plugins/data/public/expressions/expressions_service.ts

```diff
@@ -23,9 +23,9 @@ import { Ast } from '@kbn/interpreter/common';
 // this type import and the types below them should be switched to the types of
 // the interpreter plugin itself once they are ready
 import { Registry } from '@kbn/interpreter/common';
-import { Adapters } from 'ui/inspector';
+import { Adapters } from 'src/plugins/inspector/public';
```

Comment:
why are we using absolute paths in some scenarios and relative in others ?

Reply:
Because ATM you can import types with absolute paths, but not classes or functions :-(
Or at least it was, last time I tried.
Once @epixa resolves this https://github.com/elastic/kibana/issues/40446 we should be able to use this type of paths everywhere.